setup.cfg

```diff
@@ -0,0 +1,26 @@
+[metadata]
+name = wgvanitykey
+version = attr:wgvanitykey.version
```

Comment:
This version string is DRY because it's only specified once as an attribute of the root module of the package.